### DISCUSSION TITLE
Installing Chatwoot and Wordpress (and others) - Docker - Ubuntu 20.04

### DISCUSSION THREAD

**Discussion by Toninobandolero** [chatwoot/chatwoot#3797] [Self hosted] [answered]
Hello everyone, and thanks for your responses.

As i'm still practicing with all this stuff, i would like to ask you what is the best option for you to install Chatwoot with Docker, using a vps running ubuntu 20.04LTS as server, and sharing with other applications as Wordpress or Admirel as containers?

Will Nginx Proxy Manager or Traefic simplify the management with this configuration?
Should i use all these containers with only one docker-compose.yml file?

Thank you :)

**Comment by pranavrajs** [comment, marked as answer]
Here is a production deployment guide for docker https://www.chatwoot.com/docs/self-hosted/deployment/docker. You can use different docker-compose for different applications. @vishnu-narayanan can you please confirm this?

**Reply by vishnu-narayanan** [reply]
Sounds about right @pranavrajs.

As mentioned in the other discussion, just make sure you don't have any port conflicts and you should be good.  Docker will take care of the isolation.
